.eslintrc.js

```diff
@@ -100,9 +100,10 @@ module.exports = {
 
   overrides: [
     {
-      files: ['bin/**/*.js'],
+      files: ['bin/**/*.js', 'dev/new-rule/index.js'],
       rules: {
         'no-console': 'off',
+        'no-unsupported-features': 'off',
```

Comment:
I do not think we want to turn off `no-unsupported-features` in all of our bin scripts (and I'm not 100% sure why we'd want to turn it off for the new `dev/new-rule/index.js` either). 

Please revert this, and if we _do_ need an override for the new `dev/new-rule/index.js` please explain what feature is used that we are relying on, and add it in its own override section (so it does not allow us to accidentally use too new of features in the main bin script).

Reply:
reverted it. sorry about that.